nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-api/src/test/java/org/apache/nifi/web/controller/ControllerSearchServiceTest.java

```diff
@@ -191,6 +185,7 @@ public void testSearchWhenGroupIsNotAuthorized() {
         testSubject.search(searchQuery, results);
         // The authorization is not transitive, children groups might be good candidates.
         thenFollowingGroupsAreSearched(Arrays.asList(
+                PROCESS_GROUP_ROOT,
```

Comment:
This looks a little strange to my be retrospect, but the documentation says “To reduce the amount of time admins spend on authorization management, policies are inherited from parent resource to child resource”, so I think in practice the case where you cannot access the process group but you can access a resource within should not happen.